internal/cws/client.go

```diff
@@ -241,7 +241,7 @@ func (c *Client) RegisterStripeWebhook(url, owner string) (string, error) {
 	}
 	defer closeBody(resp)
 	switch resp.StatusCode {
-	case http.StatusOK:
```

Comment:
Was this from a recent change? Seems it was working with this status since the beginning of time?

Reply:
This code hadn't been run because the CWS Spinwicks were broken for a different reason